codes/classical/q-ary_digits/determinant_code.yml

```diff
@@ -0,0 +1,27 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+code_id: determinant_code
+physical: q-ary_digits
+logical: q-ary_digits
+
+name: 'Determinant Code'
+introduced: '\cite{doi:10.1109/TIT.2016.2616137}'
+
+description: |
+    Determinant Codes give optimal exact repair regenerating codes for any \([n,k,d=k]\) at all the points of the storage bandwidth trade-off curve. The codes are linear, and the 
+    exact regenerating property is provided based on fundamental properties of matrix determinants. Thus the code is called determinant code. The field size required for
+    this code construction is linear in \(n\). 
+protection: 'Corrects upto \(n-k\) erasures on coordinates.'
+
+notes:
+  - 'For exact repair, the interior points of the storage-bandwidth trade-off curve can be shown to be the convex hull of \(k\) corner points described by \((\alpha_m,\beta_m)= (\binom{k}{m},\binom{k-1}{m-1})\) for \(m=1,2,\cdots,k\).'
+#relations:
+#  parents:
+#    - code_id: regenerating_code
```

Comment:
When you submit as one pull request along with regenerating, you can uncomment this.

Reply:
regenerating